x-pack/plugins/viz_editor/common/query_types.ts

```diff
@@ -114,7 +132,7 @@ export interface OrderByOperation {
 }
 
 export interface Query {
-  index: string;
+  indexPattern: string;
```

Comment:
This probably makes sense, but technically the query can be run against any index. I wonder if that matters?

Reply:
@chrisdavies I think this is a case where our terminology is intersecting with elasticsearch terminology in a confusing way. I believe that when you search on Elasticsearch, the parameter "index" can include wildcards, and so our "index pattern name" is mapped to the field "index" when searching. Is that right? If so, how should we clarify?